test/bin/build_images.sh

```diff
@@ -545,6 +545,10 @@ build_images.sh [-iIsdf] [-l layer-dir | -g group-dir] [-t template]
 
   -d      Dry run by skipping the composer start commands.
 
+  -e      Extract container images.
+
+  -E      Do not extract container images.
+
```

Comment:
We should not need both options. Let's keep the extraction ON by default and add "do not extract" flag.